### PR TITLE
Fix APT 404 Errors on unsupported platforms.

### PR DESCRIPTION
**Pull request by asdfugil**
This fixes apt 404 errors caused by the absence of repository on unsupported platforms.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -786,7 +787,6 @@ MAKEFLAGS += --jobs=$(shell $(GET_LOGICAL_CORES))
 endif
 
 PROCURSUS := 1
-
```
Comment (Diatrus) [inline Makefile:789]: Don't remove any extra whitespace for no reason.

File: Makefile [hunk 2]
```diff
@@ -868,9 +867,12 @@ endif # $(shell [ "$(CFVER_WHOLE)" -ge 1600 ] && echo 1),1
 	touch $(BUILD_STRAP)/strap/.procursus_strapped
 	touch $(BUILD_STRAP)/strap/private/etc/apt/sources.list.d/procursus.sources
 	echo -e "Types: deb\n\
-URIs: https://apt.procurs.us/\n\
+URIs: $(MEMO_REPO_URI)\n\
 Suites: $(MEMO_TARGET)/$(MEMO_CFVER)\n\
 Components: main\n" > $(BUILD_STRAP)/strap/private/etc/apt/sources.list.d/procursus.sources
+ifneq (,$(findstring 404 Not Found,$(shell wget -qO- $(MEMO_REPO_URI)/dists/$(MEMO_TARGET)/$(MEMO_CFVER) --content-on-error || ([ "$$?" -eq 8 ] && true) || false)))
```
Comment (Diatrus) [inline Makefile:873]: Don't do this

File: Makefile [hunk 3]
```diff
@@ -895,15 +897,22 @@ else # ($(MEMO_PREFIX),)
...
 Components: main\n" > $(BUILD_STRAP)/strap/$(MEMO_PREFIX)/etc/apt/sources.list.d/procursus.sources
 else
 	echo -e "Types: deb\n\
-URIs: https://apt.procurs.us/\n\
+URIs: $(MEMO_REPO_URI)\n\
 Suites: $(MEMO_TARGET)/$(MEMO_CFVER)\n\
 Components: main\n" > $(BUILD_STRAP)/strap/$(MEMO_PREFIX)/etc/apt/sources.list.d/procursus.sources
 endif
+ifneq (,$(findstring darwin,$(MEMO_TARGET)))
+ifneq (,$(findstring 404 Not Found,$(shell wget -qO- $(MEMO_REPO_URI)/dists/$(MACOSX_SUITE_NAME)/main --content-on-error || ([ "$$?" -eq "8" ] && true) || false)))
```
Comment (Diatrus) [inline Makefile:910]: Don't do this

**Review by Diatrus — CHANGES_REQUESTED** [review]

**Comment by asdfugil** [comment]
not the most useful thing ever. closing.

**Pull request closed without merging**